### PR TITLE
Override HOST value in usaspending-api in order to allow for permalinks.

### PR DESCRIPTION
**Pull request by BrianZito**
Follows up on api changes made in https://github.com/fedspendingtransparency/usaspending-api/pull/2151

### CODE REVIEW

File: usaspending-deploy/usaspending-api-image.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@
       become: true
       git:
         repo: "{{ REPO }}"
-        version: "{{ BRANCH }}"
+        version: "mod/dev-3841-add-url-to-downloads"
```
Comment (myazidi) [inline usaspending-deploy/usaspending-api-image.yml]: Version needs to point back to "{{ BRANCH }}". We only had to hardcode your branch for testing purposes.
Also make sure you only merge when your changes for `usaspending-api` are in master.

File: usaspending-deploy/usaspending-api-image.yml [hunk 2]
```diff
@@ -65,6 +65,7 @@
       git:
         repo: git@github.com:fedspendingtransparency/data-act-build-tools.git
         dest: /data-act/build-tools
+        version: "config/permalink"
```
Comment (myazidi) [inline usaspending-deploy/usaspending-api-image.yml]: same thing here this line needs to be removed.

File: usaspending-deploy/usaspending-api-launch.yml [hunk 3]
```diff
@@ -217,6 +217,20 @@
         dest: "{{ CODE_HOME }}/usaspending_api/settings.py"
         regexp: '\s*DEBUG =.*'
         line: "DEBUG = False"
+        
+    - name: API | add host for permalink
+      when: BUILD_TYPE == 'api'
+      lineinfile:
+          dest: "{{ CODE_HOME }}/usaspending_api/settings.py"
+          regexp: '\s*HOST =.*'
+          line: "HOST = '{{ DOMAIN }}'"
```
Comment (myazidi) [inline usaspending-deploy/usaspending-api-launch.yml]: do we really need `HOST` in the API as well !!!

Comment (BrianZito) [reply usaspending-deploy/usaspending-api-launch.yml]: Yes, for the non-bulk downloads.

Comment (myazidi) [reply usaspending-deploy/usaspending-api-launch.yml]: that's right, here it's copying it to the api instance.

**Pull request merged**